### DISCUSSION TITLE
global installation & update check feature

### DISCUSSION THREAD

**Discussion by msaaddev** [directus/directus#6794] [Feature Requests]
I was reading the code of `create-directus-project` and found out that there is no option to use the CLI if it is being installed globally. I think it is intentional. But I would suggest that since people sometimes install the CLIs globally, we add an option to use `create-directus-project` locally. 

Also, this will bring an issue of what if there is a new update available for the CLI and the user is still using the old version. And to tackle it, we can use [`update-check`](https://www.npmjs.com/package/update-check) package which is also being used by `create-next-app` for notifying users if there is an update available.

**Comment by rijkvanzanten** [comment]
Sounds good to me! Wanna open a PR for that? 🙂

**Reply by msaaddev** [reply]
Sure thing
